Apps/ValidationTests/Scripts/validation_native.js

```diff
@@ -6,6 +6,13 @@ var saveResult = true;
 var testWidth = 600;
 var testHeight = 400;
 
+// Random replacement
```

Comment:
nit: If this is a common algorithm, can you add a link in a comment about where it came from? Alternatively, there's a simple random function used with the Babylon.js procedural textures (it's from a `vec2`, but could be adapted easily), so we could just use that one and reference it there. Not a big deal, just a leftover sensibility I have about keeping track of random algorithms.

Reply:
I've added a note in the validation tests infrastructure issue to refactor the validation_native.js with the bjs version. I'll take care of that rando, function at the same time.